### ISSUE TITLE
Add rubrics for every lab

### ISSUE THREAD

**Issue by lorihollasch** [TEALS-IntroCS/introduction-to-computer-science-principles#18]
Feedback from a new-to-TEALS Intro CS classroom teacher: Add rubrics for every lab, not just for projects. Rubrics are commonplace and provide students with a quick snapshot of areas they need to work on and where they excel. (School is very appreciate of the curriculum)

**Comment by Land-Of-10K** [comment]
I agree that this is necessary. Having a rubric for every lab would be helpful.
